integration-cli/docker_api_containers_test.go

```diff
@@ -138,11 +138,7 @@ func TestContainerApiStartVolumeBinds(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	bindPath, err := ioutil.TempDir(os.TempDir(), "test")
-	if err != nil {
-		t.Fatal(err)
-	}
-
+	bindPath := path.Join("/tmp", fmt.Sprintf("test.%s", makeRandomString(10)))
```

Comment:
one thing that I've been wondering about... in all of my windows programming I've always been able to use / instead of \\ in my paths - is this not true for golang? Just wondering if we really need to use filepath.Join() when we're on windows at all.